app/CL_EditorImpl/src/main/java/org/jjazz/cl_editorimpl/CL_EditorImpl.java

```diff
@@ -1436,6 +1477,10 @@ private void addItem(int barIndex, ChordLeadSheetItem<?> item)
             // Listen to sectionOnNewLine changes
             cliSection.getClientProperties().addPropertyChangeListener(this);
         }
+        if (item instanceof IR_DisplayTransposable transposableItem)
```

Comment:
error here: it's the ItemRenderer which is transposable, your test should be inside the first loop (ir)